pants

```diff
@@ -205,11 +209,8 @@ pants_dir="$(bootstrap_pants "${pants_version}" "${python}")"
 # See https://blog.phusion.nl/2017/10/13/why-ruby-app-servers-break-on-macos-high-sierra-and-what-can-be-done-about-it/
 export no_proxy='*'
 
-if [[ -z "${pants_config_file}" ]]; then
-  pants_config_file_flag="";
-else
-  pants_config_file_flag=--pants-config-files="[\"${PWD}/${pants_config_file}\"]";
+if ! [[ -v PANTS_CONFIG_FILES ]] && [[ -n ${pants_config_file} ]]; then
+  export PANTS_CONFIG_FILES="[\"${PWD}/${pants_config_file}\"]";
```

Comment:
Should this be a plus? What happens if a user appends via the CLI?

Reply:
The CLI value will be appended to this.

We want to override the default here, so I don't think we want a +.